src/main/java/com/endava/endabank/constants/Routes.java

```diff
@@ -0,0 +1,28 @@
+package com.endava.endabank.constants;
+
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public final class Routes {
```

Comment:
Hi,

You can change this class for an interface as :

`public interface Routes1 {
    String API_ROUTE = "/api/v1";
    String USERS_ROUTE = "/users";
    String API_USERS_ROUTE = API_ROUTE + USERS_ROUTE;
    String LOGIN_ROUTE = "/login";
    String API_LOGIN_ROUTE = API_ROUTE + LOGIN_ROUTE;
    String FRONTEND_ROUTE = "http://localhost:3000";
    String RESET_PASSWORD_ROUTE = "/reset-password";
    String APPROVE_ACCOUNT_ROUTE = "/approve/{id}";
    String RESET_PASSWORD_FRONTEND_ROUTE = FRONTEND_ROUTE + "/reset-password/?token=";
    String UPDATE_PASSWORD = "/change-password";
    String SWAGGER_IU = "/swagger-ui.html/**";
    String SWAGGER_JSON = "/v2/api-docs/**";
    String CONFIGURATION = "/configuration/**";
    String SWAGGER = "/swagger*/**";
    String WEB_JARS = "/webjars/**";
    String RESOURCE_DETAILS = "/details";
    String EMAIL_VALIDATION_FRONTEND_ROUTE = FRONTEND_ROUTE + "/verify-email/?token=";
    String EMAIL_VALIDATION_ROUTE =  "/verify-email";
    String SECRET_JWT_DEFAULT = "secret";
}`